examples/lines_bars_and_markers/timeline.py

```diff
@@ -0,0 +1,68 @@
+"""
+===============================================
+Creating a timeline with lines, dates, and text
+===============================================
+
+How to create a simple timeline using Matplotlib release dates.
+
+Timelines can be created with a collection of dates and text. In this example,
+we show how to create a simple timeline using the dates for recent releases
+of Matplotlib. First, we'll pull the data from GitHub.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import urllib.request
+import json
+
+# Grab a list of Matplotlib releases
+url = 'https://api.github.com/repos/matplotlib/matplotlib/releases'
+data = json.loads(urllib.request.urlopen(url).read().decode())
```

Comment:
I'm not sure if the examples should depend on web links. Or is this safe to link to github.com because the tests themselves run on github?